### PR TITLE
Virtual and funding guides

### PR DESCRIPTION
**Pull request by braden-holgate**
Added a separate Funding Card Spending Guide. Funding and card issuance are loosely coupled and can be managed independently. The guides reflect this and it makes for much cleaner reading and diagramming.

The get-prerequisites endpoint has been removed and redirected to the new endpoint.

***Screenshots of Diagrams***
Card issuing sequence
Screenshot 2023-09-04 at 2 26 19 PM

Funding diagram
Screenshot 2023-09-04 at 2 26 50 PM

Fetching secure card details:

Screenshot 2023-09-04 at 2 27 06 PM
[Ticket Link: _link_here_](https://www.notion.so/immersve/Create-new-wallet-integrator-guide-8d9657798cdf43079e541a903ae09e0f)

### CODE REVIEW

File: docs/guides/authentication.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Each request must be made with the following headers:
 
 ## SIWE Authentication
 
+The use of web3-native authentication mechanisms within the context of card issuance mean that the same level of protection to a user's funds in self-custodial wallets is applied to their activities as a cardholder.
```
Comment (rb455) [inline docs/guides/authentication.md:28]: meanS

File: docs/guides/fetching-secure-card-information.mdx [hunk 2]
```diff
@@ -1,15 +1,32 @@
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Fetching secure card information
+# Fetching Secure Card Information
+
 
 In order to retrieve the secure details of a card (cvv, expiry, pan), a one-time token must be generated.
```
Comment (catluin) [inline docs/guides/fetching-secure-card-information.mdx:7]: ✏️ Should we write `CVV` and `PAN` in capitals? I think those are abbreviations

File: openapi/endpoints/cards/models/create-card-async-request.yaml [hunk 3]
```diff
@@ -8,4 +8,4 @@ properties:
     description: Id of card program to use when creating a card
   fundingSourceId:
     type: string
-    description: Id of the funding source the card will authorize against
\ No newline at end of file
+    description: Id of the Funding Source the card will authorize against
```
Comment (catluin) [inline openapi/endpoints/cards/models/create-card-async-request.yaml:11]: ✏️ capitalize `ID` for consistency?

Comment (braden-holgate) [reply openapi/endpoints/cards/models/create-card-async-request.yaml:11]: Thought I got em all!

**Review by ncjones — APPROVED** [review]

**Pull request merged**